hydradx/model/indexer_utils.py

```diff
@@ -203,6 +192,24 @@ def get_omnipool_liquidity(
     return liquidity, hub_liquidity
 
 
+def get_current_block_height():
```

Comment:
I had not made this function yet because I'm not convinced it will be identical for every indexer we query. Maybe we should make the indexer an input? but then I'm not sure it'd be the same query for the different indexers?

When we are analyzing data from one of the storage indexers, we are probably going to want to know what the latest block number in the indexer in question was, not just the unified-prod indexer.